resources/eventing/charts/nats/templates/40-cr.yaml

```diff
@@ -22,3 +22,14 @@ spec:
   natsConfig:
     debug: true
     trace: true
+  template:
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
"preferredDuringSchedulingIgnoredDuringExecution" will distribute the 3 nats instances on the 2 Kyma nodes (in a sdandard SKR installation). But during the draining of a node (caused by the upgrade of K8S version), all the 3 Nats instances are migrated to the only one node which is active.
 
After a new node is attaching  cluster, the active 3 Nats instances are NOT redistributed again between now the two active node but are remaining running on one node.
It seems that we have to find a better appraoch, but the fix is ok for now.